prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -76,27 +100,26 @@ class LivdTableDownload() : CliktCommand(
         //
         URL(livdFileUrl).openStream().use { input ->
             if (outputFile.exists()) {
-                val c = prompt("$outputFile file is already existed: You want to overwrite it (y/n)?", "n")
+                val c = prompt("\t$outputFile file is already existed: You want to overwrite it (y/n)?", "n")
                 if (c?.lowercase() == "n") {
-                    return false
+                    return ""
                 } else {
-                    print("The $outputFile file is overwriting.\n")
+                    TermUi.echo("\tThe $outputFile file is overwriting.")
                 }
-            } else {
-                print("The $outputFile file is downloaded.\n")
             }
 
             try {
                 FileOutputStream(outputFile).use { output ->
                     input.copyTo(output)
                 }
             } catch (e: Exception) {
-                print("Error: $e")
-                return false
+                TermUi.echo("\tERROR: $e")
```

Comment:
These errors do not report an error back to the shell.  In the next issue, change all these error printouts to correctly return an error.  There are examples in some other CLI classes